exampleCourse/questions/templateRandomIntegerInput/info.json

```diff
@@ -0,0 +1,7 @@
+{
+    "uuid": "5E64666F-1588-4EBB-8E8D-C1FA4141F20D",
```

Comment:
this uuid is conflicting with `templateIntegerInput` and that is why travis is not happy :slightly_smiling_face:

Reply:
Interesting.... I thought `templateRandomIntegerInput` was deleted (or actually renamed as  `templateIntegerInput`.